material-overrides/assets/stylesheets/tanssi.css

```diff
@@ -1342,4 +1342,57 @@ h2.subtitle.mobile-subtitle {
   border-radius: 1em;
   vertical-align: baseline;
   box-shadow: none;
+}
+
+/* Cards Styling */ 
+.md-typeset .grid.cards>ul>li {
+  border: 1px solid #FFFAFA;
+  background-color: #3D3D3D;
+}
+.md-typeset .grid.cards>ul>li:hover {
+  border: 1px solid #189A9A;
+  background-color: #1B1B1B;
+}
+.md-typeset a:hover {
+  transform: translateX(10px);
+  transition: 0.3s ease;
+}
+.grid.cards > ul > li:hover >p:first-child {
+  transform: translateX(10px);
+  transition: 0.3s ease;
+}
+.md-typeset hr {
+border: o solid 
+}
+.md-typeset .grid {
+  grid-gap: 1rem;
+}
+.md-typeset .grid {
+  display: grid;
+  grid-template-columns: repeat(auto-fit, minmax(min(100%, 16rem), 1fr));
+    margin: 1em 0;
+}
+.md-typeset .grid.cards > ul > li:hover >p:first-child .twemoji svg {
+  fill:  #189A9A;
+}
+.md-typeset .grid.cards>ol>li>:last-child,
+.md-typeset .grid.cards>ul>li>:last-child,
+.md-typeset .grid>.card>:last-child {
+  margin-top: auto;
+}
+.md-typeset .grid.cards>ol>li,
+.md-typeset .grid.cards>ul>li,
+.md-typeset .grid>.card {
+  display: flex;
+  flex-direction: column;
+}
+.md-typeset .grid.cards>ul>li:hover >p:last-child{
+  transform: translateX(10px);
+  transition: 0.3s ease;
+}
```

Comment:
Since this styling is the same styling used for `.grid.cards > ul > li:hover >p:first-child {` (currently on line 1360), we should combine them, so:

```
.grid.cards > ul > li:hover >p:first-child,
.md-typeset .grid.cards>ul>li:hover >p:last-child{
  transform: translateX(10px);
  transition: 0.3s ease;
}
```

Reply:
Actually, also just noticed the same styling is used for `.md-typeset a:hover` on line 1356. So ideally, these are all combined